cmd/collection/main.go

```diff
@@ -81,8 +81,6 @@ func main() {
 				"maximum per-transaction gas limit")
 			flags.BoolVar(&ingestConf.CheckScriptsParse, "ingest-check-scripts-parse", true,
 				"whether we check that inbound transactions are parse-able")
-			flags.BoolVar(&ingestConf.AllowUnknownReference, "ingest-allow-unknown-reference", true,
```

Comment:
Removed this option for collection nodes because we need to be able to resolve the epoch w.r.t. the reference block in order to select a mempool for the transaction. Currently, even with this set to true, the `ingest` engine will throw an error when it attempts to query the epoch.